lib/elixir_conf_africa/events.ex

```diff
@@ -21,6 +21,22 @@ defmodule ElixirConfAfrica.Events do
     Repo.all(from e in Event, order_by: [desc: e.id])
   end
 
+  def get_elixir_conf_event_and_ticket_types do
+    get_elixir_conf_event()
+    |> Repo.preload(:ticket_types)
+  end
+
+  defp get_elixir_conf_event do
+    Repo.get_by(Event, name: "ElixirConf Africa")
+  end
+
+  def get_all_available_tickets do
+    get_elixir_conf_event_and_ticket_types()
+    |> Map.get(:ticket_types)
+    |> Enum.map(fn x -> x.number end)
+    |> Enum.sum()
+  end
```

Comment:
I believe this can be fixed by `WindowAPI` function count/0 [count](https://hexdocs.pm/ecto/Ecto.Query.API.html#count/0)

Reply:
@okothkongo  , Can we use count to give us the sum or just the number of records , kindly assist in this

Reply:
Sorry i meant [sum](https://hexdocs.pm/ecto/Ecto.Query.API.html#sum/1)

Reply:
sorry I meant [sum](https://hexdocs.pm/ecto/Ecto.Query.API.html#sum/1)

Reply:
Thanks for this , it is working